joblib/__init__.py

```diff
@@ -71,9 +71,6 @@
              [ 4.,  2.,  1.]]))
       ___________________________________________________________square - 0...s, 0.0min
 
-      >>> c = square(a)
-      >>> # The above call did not trigger an evaluation
```

Comment:
I was seeing how this doctest behaves in absence of numpy. I modified parts of it temporarily but I missed these two lines in reverting back. Sorry I'll restore them.